MdeModulePkg/Library/DxeCapsuleLibFmp/DxeCapsuleProcessLib.c

```diff
@@ -180,8 +180,11 @@ UpdateImageProgress (
   }
 
   Status = DisplayUpdateProgress (Completion, Color);
+  if (EFI_ERROR(Status)) {
+    DEBUG((DEBUG_WARN, "DisplayUpdateProgress returned %r\n", Status));

```

Comment:
Nit: Add a space after DEBUG and the open parenthesis.

Reply:
done.

Reply:
Thanks!